source/mainnet/net/references/multi-sig.rst

```diff
@@ -0,0 +1,448 @@
+
+   Your ``browser_wallet.export`` file should look similar to the following browser wallet export template:
+
+   .. code-block:: json
+      :force:
+
+      {
+         "type": "concordium-browser-wallet-account",
+         "v": 0,
+         "environment": "testnet",
+         "value": {
+            "accountKeys": {
+               "keys": {
+                  "0": {
+                     "keys": {
+                        "0": {
+                           "signKey": "<Key_0_Private_Key_Without_0x_Prefix>",
+                           "verifyKey": "<Key_0_Public_Key_Without_0x_Prefix>"
+                        },
+                        "1": {
+                           "signKey": "<Key_1_Private_Key_Without_0x_Prefix>",
+                           "verifyKey": "<Key_1_Public_Key_Without_0x_Prefix>"
+                        }
+                     },
+                     "threshold": 2
+                  }
+               },
+               "threshold": 1
+            },
+            "credentials": {
+               "0": "97f325c9f86066ab0c80ff879c21629eb67818841940869308d6a72886d18f8668e62e43ad228fdcbda245d0722454df"
+            },
+            "address": "4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix"
+         }
+      }
+
+   You can import the keys of this ``browser_wallet.export`` file and associate it to your account as follows:
+
+   .. code-block:: console
+
+      $concordium-client config account import browser_wallet.export --name <choose-a-name-for-your-account>
+
+.. dropdown:: Assign an already imported key to your account
+
+   If you already imported the new key to ``concordium-client`` but it is associated with a different account
+   (e.g. because the public-private key pair was generated by creating a new account on Concordium which was imported
+   to ``concordium-client`` and this key pair was re-used in your multi-sig account),
+   you can look up the key in the local key directory of the ``concordium-client`` tool and associate
+   the file's content to your multi-sig account.
+
+   .. note::
+
+      We do not recommend to re-use keys on different accounts in production.
+
+   .. code-block:: console
+
+      $concordium-client config account update-keys --keys new-keys.json --account <account-name-or-address>
+
+   where ``new-keys.json`` is a file of the following format:
+
+   .. code-block:: json
+      :force:
+
+      {
+         "cidx": {
+            "kidx": {
+               "encryptedSignKey": {
+                  "metadata": {
+                     "encryptionMethod": "AES-256",
+                     "iterations": ...,
+                     "salt": ...,
+                     "initializationVector": ...,
+                     "keyDerivationMethod": "PBKDF2WithHmacSHA256"
+                  },
+                  "cipherText": ...
+               },
+               "verifyKey": ...,
+               "schemeId": "Ed25519"
+            },
+            ...
+         },
+         ...
+      }
```

Comment:
How should the user get the values for this file? I feel like this would be confusing to follow (at least I don't understand what you're supposed to do to get to this point).

Reply:
I updated the text and gave a better example.

Reply:
I still don't think it's entirely clear how the user will actually get the values. It would be helpful to explain where the directory is on the host machine. Or is it possible to print these values somehow?